module/VuFindConsole/src/VuFindConsole/Command/Util/ExpireOaiResumptionCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Console command: expire OAI resumption tokens.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2020.
+ * Copyright (C) The National Library of Finland 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Console
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFindConsole\Command\Util;
+
+use DateTime;
+use DateTimeZone;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Input\InputArgument;
+
+/**
+ * Console command: expire OAI resumption tokens.
+ *
+ * @category VuFind
+ * @package  Console
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+#[AsCommand(
+    name: 'util/expire_resumption_tokens'
+)]
+class ExpireOaiResumptionCommand extends AbstractExpireCommand
+{
+    /**
+     * Help description for the command.
+     *
+     * @var string
+     */
+    protected $commandDescription = 'Database oai_resumption table cleanup';
+
+    /**
+     * Label to use for rows in help messages.
+     *
+     * @var string
+     */
+    protected $rowLabel = 'resumption tokens';
+
+    /**
+     * Minimum legal age (in days) of rows to delete.
+     *
+     * @var int
+     */
+    protected $minAge = 0;
+
+    /**
+     * Configure the command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        parent::configure();
+        $arguments = $this->getDefinition()->getArguments();
+        $arguments['age'] = new InputArgument(
+            'age',
+            InputArgument::OPTIONAL,
+            'Minimum age (IGNORED in this command)',
```

Comment:
Is there a reason not to support this? I understand we wouldn't normally use it, but would it actually hurt anything to have it for consistency's sake?

Reply:
Makes sense. In that case, how would you feel about refactoring the code that adds the age argument in `AbstractExpireCommand::configure()` to a `protected addAgeArgument()` method so you can disable it entirely in this subclass? That would be less confusing for the end user than an argument that says not to use it.